homeassistant/components/lg_thinq/vacuum.py

```diff
@@ -114,7 +114,7 @@ def _update_status(self) -> None:
         super()._update_status()
 
         # Update state.
-        self._attr_state = ROBOT_STATUS_TO_HA[self.data.current_state]
+        self._attr_state = ROBOT_STATUS_TO_HA.get(self.data.current_state)
```

Comment:
Hmm, odd. What does that mean when this happens? Is the vacuum offline or unavailable?

Reply:
You are correct.
if vacuum is offline, because of wifi disconnection or something else, this could happen.

Reply:
In that case, this PR isn't correct.

If an device isn't available, the `available` property of an entity should be set to `False`.

The current proposed change in this PR will cause the entity to become `unknown`, but it is known... we know it isn't available :)

../Frenck

Reply:
Got it. Let us drop this one.